src/main/java/org/jetbrains/bsp/bazel/server/bsp/resolvers/actiongraph/ActionGraphV1Parser.java

```diff
@@ -11,15 +11,15 @@
 import org.jetbrains.bsp.bazel.commons.Uri;
 
 // TODO(illicitonion): Index, cache, etc
-public class ActionGraphParser {
+public class ActionGraphV1Parser implements ActionGraphParser {
 
   private final AnalysisProtos.ActionGraphContainer actionGraph;
 
-  public ActionGraphParser(AnalysisProtos.ActionGraphContainer actionGraph) {
+  public ActionGraphV1Parser(AnalysisProtos.ActionGraphContainer actionGraph) {
     this.actionGraph = actionGraph;
   }
 
-  public List<String> getInputs(String target, List<String> suffixes) {
+  private List<String> getInputs(String target, List<String> suffixes) {
```

Comment:
I think that this method should be under the `getInputsAsUri` method

Reply:
+ it would be nice to add `@Override` annotation for `getInputsAsUri` and `getOutputs`